Aptoide/aptoide-client-v8
# [Feature/ASV-1478] AppView wallet install offer

**What does this PR do?**

   Shows a promotion offer of the wallet when an user goes into an App with APPC (billing or campaign)

**Database changed?**

   No

**Where should the reviewer start?**


**How should this be manually tested?**

Promotion is shown if promotion type is active and wallet is not claimed yet. Check ticket & confluence documentation to view the different states.

  Flow on how to test this or QA Tickets related to this use-case: [confluence doc](https://aptoide.atlassian.net/wiki/spaces/AN/pages/1134559324/APPC+App+Wallet+Offer)

**What are the relevant tickets?**

  Tickets related to this pull-request: [ASV-1478](https://aptoide.atlassian.net/browse/ASV-1478)

**Code Review Checklist**
